alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -905,16 +908,21 @@ impl Canvas {
                 let y = y as f32;
                 let x = x as f32;
                 let d = (x * x + y * y).sqrt() + d_bias;
-                let v = if d < r - w - 1.0 {
-                    0.0
+                let v = if d < r - w - 1. {
+                    // Inside the ring
```

Comment:
```suggestion
                    // Inside the circle.
```

Comments have dots at the end and generally speaking it's called `circle`, in that case.

Also, given that `d_bias` is the center of the `circle`, I think the `inside` and `outer` are actually swapped?